dotnetv3/dynamodb/FromSQL/UpdateItemDataModelExample/UpdateItemDataModelExample/Entry.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+
+namespace UpdateItemDataModelExample
+{
+    using Amazon.DynamoDBv2.DataModel;
+
+    // snippet-start:[dynamodb.dotnet35.UpdateItemDataModel.Entry]
+
+    /// <summary>
+    /// If you change the table name elsewhere (as in an app.config in
```

Comment:
confusing. Suggest 'If you change the table name elsewhere (for example, in an app.config in another project)

Reply:
Updated the text.